src/ts/ssh-tcp/tcpListenerFactory.ts

```diff
@@ -10,11 +10,12 @@ export interface TcpListenerFactory {
 	 * Creates and starts a TCP listener for the specified local network address and port
 	 * number.
 	 *
+	 * @param remotePort The remote port that this local port will connect to (if known).
 	 * @param localIPAddress Local IP address to listen on.
 	 * @param localPort Requested local port to listen on, or 0 to use a random
 	 * available port number.
-	 * @param canChangePort True if the factory is allowed to select a different
-	 * port number than the one that was requested; if false then the factory must either
+	 * @param canChangeLocalPort True if the factory is allowed to select a different
```

Comment:
The parameter name is still `canChangePort` below.

Reply:
Fixed.